packages/web/e2e/uploadTrack.test.ts

```diff
@@ -222,8 +222,8 @@ test('should upload a premium track', async ({ page, browser }) => {
   const newPage = await openCleanBrowser({ browser })
   const buyButton = newPage.getByRole('button', { name: /buy/i })
   newPage.goto(trackUrl)
-  await expect(buyButton).toBeVisible()
-  await expect(newPage.getByText('$' + price)).toBeVisible()
+  await expect(buyButton).toBeVisible({ timeout: 20000 })
+  await expect(newPage.getByText('$' + price)).toBeVisible({ timeout: 20000 })
```

Comment:
nit: fwiw I think if the buy button is visible, the price ought to be visible within 10 seconds

Reply:
ah so i only need the first timeout and it'll wait for 20s before calling the next expect? gotcha